src/Config/CliBuilder.ts

```diff
@@ -89,12 +94,33 @@ export class CliBuilder {
       .reduce((acc: Argv, item: CommandModule) => {
         const handler = item.handler.bind(item);
 
-        item.handler = (args) =>
-          runWithAsyncContext(() => {
-            setContext('args', args);
+        item.handler = async (args: Arguments) => {
+          const systemConfigReader = new SystemConfigReader(args.api as string);
+          const systemConfig = await systemConfigReader.read();
 
-            handler(args);
+          initSentry({
+            attachStacktrace: true,
+            dsn: systemConfig.sentryDsn,
+            release: process.env.VERSION,
+            beforeSend: (event) => {
```

Comment:
Avoid implicit any

Reply:
It's not implicit any, it's implicit `ErrorEvent`

Reply:
Yes, but I don't see any problem here. I would say it's matter of approach whether to annotate types or use type inference.

One comment above you suggested to use shorter syntax for `beforeSend: (event) => {`, which is good. So why to have more code in this case? Explicit `ErrorEvent` says nothing and protects from nothing.

So I would keep it short

Reply:
> Explicit ErrorEvent says nothing and protects from nothing.

Only if you are going to incorporate the following changes: https://github.com/NeuraLegion/bright-cli/pull/466#discussion_r1346841142. Otherwise, the arrow function can be extracted to a variable, and then contextual type inference won't work. You would have to declare type anyway.

Reply:
They are already incorporated. But even if they wouldn't the type would be still properly inferred. 

The case when an arrow function would be assigned to a variable is irrelevant since it's just different code, which won't even compile because of enabled strict mode.